PlanGrid.Api/PlanGridHttpHandler.cs

```diff
@@ -79,12 +81,18 @@ public PlanGridHttpHandler(string accessToken, RefitSettings settings, string ve
             throw new FailedRequestException(HttpStatusCode.ServiceUnavailable, $"Service unavailable after retrying {maxRetries} times.");
         }
 
-        private string BuildAuthenticationToken(string accessToken)
+        private string BuildAuthenticationToken(string accessToken, TokenType tokenType)
         {
-            string unencoded = $"{accessToken}:";
-            byte[] authParamBytes = Encoding.ASCII.GetBytes(unencoded);
-            string encodedAuthParams = Convert.ToBase64String(authParamBytes);
-            return encodedAuthParams;
+            if (tokenType == TokenType.Basic)
```

Comment:
This is fine, but a `switch` statement might be more idiomatic for this purpose.